parts/kubernetesagentcustomdata.yml

```diff
@@ -107,7 +107,7 @@ write_files:
     DOCKER_OPTS=
     CUSTOM_CMD=/bin/true
     KUBELET_REGISTER_SCHEDULABLE=true
-    KUBELET_NODE_LABELS=role=agent
+    KUBELET_NODE_LABELS=role=agent{{ GetKubernetesCustomLabels . }}
     KUBELET_POD_INFRA_CONTAINER_IMAGE={{WrapAsVariable "kubernetesPodInfraContainerSpec"}}
```

Comment:
Any good reason for not including the default `role=agent` label in the `GetKubernetesCustomLabels` function?   If we did that, then it might be more accurate to rename the function to `GetKubernetesLabels` and which would provide the default and any additional custom labels provided by `profile.CustomNodeLabels`.